airbyte-webapp/src/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+import styled from "styled-components";
+
+const Container = styled.div`
+  * {
```

Comment:
@isalikov aren't you changing here styles for all elements instead of Markdown specific ones?